client/src/js/bundle.js

```diff
@@ -0,0 +1,4 @@
+import 'bootstrap/js/dist/util';
+import 'bootstrap/js/dist/tooltip';
+
+window.jQuery('body').tooltip({selector: '[data-toggle=tooltip]'});
```

Comment:
not ideal - though just leave as is - we can update this with follow up pr here https://github.com/creative-commoners/silverstripe-login-forms/pull/1/files#diff-6fb8efb7c9bd3ed9c20610108631b6751c85ef17ac1cf0ed7e31bda800ed3c32R5

Reply:
How would you like to see this done?
This is how we do it in `admin` through a `customJavascript` call.

Reply:
No need to update - I'm currently working through this here https://github.com/silverstripe/silverstripe-login-forms/pull/89/files#diff-6fb8efb7c9bd3ed9c20610108631b6751c85ef17ac1cf0ed7e31bda800ed3c32R5